config/aches.json

```diff
@@ -8,7 +8,7 @@
   "extrovert": {"title": "Extrovert", "description": "finish a trade with someone", "is_hidden": false, "category": "Cat Hunt"},
   "fast_catcher": {"title": "Fast Catcher", "description": "catch a cat in under 5 seconds", "is_hidden": false, "category": "Cat Hunt"},
   "slow_catcher": {"title": "Slow Catcher", "description": "catch a cat in over a hour", "is_hidden": false, "category": "Cat Hunt"},
-  "collecter": {"title": "Collecter", "description": "get all cat types", "is_hidden": true, "category": "Cat Hunt"},
+  "collector": {"title": "Collector", "description": "get all cat types", "is_hidden": true, "category": "Cat Hunt"},
```

Comment:
changing id removes it from every single existing user not to mention requires a db migration